test/data/testconfig-apibinder-offline.json

```diff
@@ -0,0 +1 @@
+{"applicationName":"supertestrpi3","applicationId":78373,"deviceType":"raspberrypi3","userId":1001,"username":"someone","appUpdatePollInterval":3000,"listenPort":2345,"vpnPort":443,"apiEndpoint":"http://0.0.0.0:3000","vpnEndpoint":"vpn.resin.io","registryEndpoint":"registry2.resin.io","deltaEndpoint":"https://delta.resin.io","pubnubSubscribeKey":"foo","pubnubPublishKey":"bar","mixpanelToken":"baz","apiKey":"boo","version":"2.0.6+rev3.prod","supervisorOfflineMode":true}
```

Comment:
You could  also test that the supervisor is in offline mode with a minimal config.json: `{ "deviceType": "raspberrypi3" }` (i.e. there's no apiEndpoint so the supervisor infers it's in offline mode).